models/eqV2_tracegrad/test_fairchem_kappa.py

```
@@ -0,0 +1,395 @@
+                print(
+                    f"Combined results saved to {save_dir / 'kappa-103-FIRE.json.gz'}"
+                )
+
+                with open(
+                    save_dir / "test_metrics.json", mode="w", encoding="utf-8"
+                ) as file:
+                    json.dump(test_metrics, file, ensure_ascii=False, indent=4)
+
+
+def run_kappa(
+    checkpoint_path: Annotated[Path, typer.Option(help="路径，包含 checkpoint.pt")],
```

Ruff (0.14.2)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)